ironfish/src/mining/poolShares.ts

```diff
@@ -119,7 +124,12 @@ export class MiningPoolShares {
     const balance = await this.rpc.getAccountBalance({ account: this.accountName })
     const confirmedBalance = BigInt(balance.content.confirmed)
 
-    const payoutAmount = BigIntUtils.divide(confirmedBalance, this.balancePercentPayout)
+    let payoutAmount: number
+    if (this.balancePercentPayoutFlag !== undefined) {
```

Comment:
Trivial, but `!this.balancePercentPayoutFlag` is idiomatic

Reply:
I wanted to possibly account for a percentage of 0. Probably not a real use case but just to be more explicit